Makefile

```diff
@@ -146,6 +168,25 @@ run_ctx_ansible_test: ci_ctx ## Run molecule check in a container
 		-e ANSIBLE_REMOTE_TMP=/tmp \
 		${CI_CTX_NAME} bash -c "make ansible_test_nodeps" ;
 
+.PHONY: run_ctx_architecture_test
+run_ctx_architecture_test: ## Run architecture_test in a container
+	$(call check-var-defined,SCENARIO_NAME)
```

Comment:
Maybe alternatively the 'check-var' could be splitted into separate first 'dependency', and then ci_ctx target can be also as explicit dependency?

e.g.:

```
## Run architecture_test in a container
.PHONY: run_ctx_architecture_test
run_ctx_architecture_test: checkvars_architecture_test ci_ctx
    podman run --rm ...
```

Reply:
Possibly (if the install_yamls is act not needed here - as its not above) the `checkvars_architecture_test` target could be then reused by both `architecture_test_nodeps` and `run_ctx_architecture_test`